assets/styl/_settings.styl

```diff
@@ -193,7 +193,117 @@ $colors ?= {
 // $contrast-low:  3
 // $contrast-high: 7
 //
-// Settings.Contrast
+// Styleguide: Settings.Contrast
 $contrast ?= 4.5
 $contrast-low ?= 3
 $contrast-high ?= 7
+
+// CSS Collections
+//
+// Colette provides `$css-main` and `$css-critical` collections to let you define:
+// - **into `$css-main`**: which styles you want to include into your final css file
+// - **into `$css-critical`**: which styles must be inlined into your html file
+//
+// By default, `$css-main` includes all Colette's files, whereas `$css-critical` is empty.
+// You can override both of them in your project stylus file.
+//
+// **Usage example of `$css-main` **
+//
+// ```
+// /* my_stylus_file.styl */
+//
+// $css-main = {
+//    'base-fonts' : '_base/_fonts',
+//    'base-form-elements' : '_base/_form-elements',
+//    'base-normalize' : '_base/_normalize',
+//    'base-typography' : '_base/_typography',
+//    'layout-grid' : '_layout/_grid',
+//    'modules-alerts' : '_modules/_alerts',
+//    'modules-teasers' : '_modules/_teasers',
+//    'theme-skin' : '_theme/_skin'
+// }
+//
+// @require `colette.styl` /* will only '@require' files listed in '$css-main */
+//
+// /* my custom styles */
+// ```
+//
+//
+// **Usage example of `$css-critical`**
+//
+// You need to create a specific stylus file to handle your critical css.
+// ```
+// /* my_criticalcss_file.styl */
+//
+// $css-critical = {
+//    'modules-avatar' : '_modules/_avatar',
+//    'modules-breadcrumb' : '_modules/_breadcrumb',
+//    'modules-buttons' : '_modules/_buttons',
+//    'modules-socialbar' : '_modules/_socialbar',
+//    'modules-tabs' : '_modules/_tabs',
+// }
+//
+// /* @require files */
+// for name, file in $css-critical
+//     @require file
+//
+// /* => will compile to my_criticalcss_file.css */
+// ```
+//
+// Then you'll need to compile the css and inline it into your html markup.
+// Suppose you have a gulpfile in your project, and work with Twig templates,
+// you can create a `gulp` task to compile the css:
+//
+// ```
+// /* gulpfile.js */
+// gulp.task('criticalcss', function () {
+//    return gulp.src(cfg.cssDir + 'loop.styl')
+//    .pipe(stylus())
+//    .pipe(gulp.dest('css/'))
+// })
+// ```
+//
+// and then simply include the compiled css file into your template
+// (dont't forget the `<style>` tag aroud your `include`!):
+// ```
+// <style>
+//    {%  include 'css/my_criticalcss_file.css' %}
+// </style>
+// ```
+//
+// Styleguide: Settings.CssCollections
+$css-main ?= {
+    'mixins-contrast-check' : '_mixins/_contrastCheck'
```

Comment:
mixins do not affect final css so we can always include it.
Same things for settings.